.github/workflows/pkgci_test_riscv64.yml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Workflow for RISCV64 cross-compilation and test jobs.
+# The concurrency of this workflow is controlled by the caller's job.
+
+name: PkgCI Test RISCV64
+
+on:
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+      write-caches:
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+      write-caches:
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  cross_compile:
+    runs-on: ubuntu-20.04
+    env:
+      PACKAGE_DOWNLOAD_DIR: ${{ github.workspace }}/.packages
+      VENV_DIR: ${{ github.workspace }}/.venv
+      PLATFORM: linux
+      ARCH: riscv_64
+      ABI: lp64d
+      TARGET_BUILD_DIR: build-linux-riscv_64
+      DOCKER_IMAGE: "gcr.io/iree-oss/emscripten@sha256:2dd4c52f1bb499ab365aad0111fe5538b685d88af38636b409b0cf6a576ab214"
+      IREE_READ_REMOTE_CCACHE: 0
+      IREE_WRITE_REMOTE_CCACHE: 0
+      IREE_READ_LOCAL_CCACHE: 1
+      IREE_WRITE_LOCAL_CCACHE: ${{ inputs.write-caches }}
+      BUILD_PRESET: test
```

Comment:
I removed "build presets" from the scripts, so this can be removed
https://github.com/iree-org/iree/blob/main/build_tools/cmake/build_riscv.sh

Reply:
Fixed it.